hoot-core/src/main/cpp/hoot/core/elements/NodeMap.h

```diff
@@ -35,7 +35,7 @@
 namespace hoot
 {
   class Node;
-  typedef HashMap<long, NodePtr > NodeMap;
+  typedef HashMap<long, NodePtr> NodeMap;
```

Comment:
Wait a minute, we have an entire header file dedicated to one `typedef`?  Above you also use a `HashMap` of `WayPtr` and `RelationPtr` objects, maybe we should add in two more `typedef`s for those too?  That is up to you.  But you'll also want to update the copyright for this file.

Reply:
Nevermind, we have a separate header file for `WayMap` and another separate file for `RelationMap`.  Disregard this one.

Reply:
Maybe I should tidy up that extra space in those files too...